test/extended/testdata/build-extended/bc-scripts-by-url.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-url"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git"
+      }
+    },
+    "strategy": {
+      "sourceStrategy": {
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "wildfly:10.0",
+          "namespace": "openshift"
+        },
+        "runtimeImage": {
+          "kind": "DockerImage",
+          "name": "phpcoder/base-jdk:8"
+        },
+        "runtimeArtifacts": [
+          {
+            "sourcePath": "/opt/s2i/destination/src/target/hello.jar"
+          }
+        ],
+        "env": [
+          {
+            "name": "USING_ENV_FROM_BUILD_CONFIG",
+            "value": "yes"
+          }
+        ],
+        "scripts": "https://raw.githubusercontent.com/php-coder/java-maven-hello-world/s2i/.s2i/bin"
```

Comment:
ok i see, you're building the master branch but pointing to remote url scripts that exist in a different branch.  so that's ok, but i'd still rename the "s2i" branch to something that better reflects what you're doing here.